### DISCUSSION TITLE
How to add a label to ActionGroup

### DISCUSSION THREAD

**Discussion by martpet** [adobe/react-spectrum#3674] [Q&A] [answered]
I want to add a label to an `ActionGroup` that looks like the labels of many other components like `LabeledValue`, `ProgressBar`, `Meter`, `Picker`, `ComboBox`, `TextField`, `TextArea`, `Slider`, `SearchField`, `RangeSlider`, `RadioGroup`, `NumberField`, `CheckboxGroup`, `CheckboxGroup`, `DateRangePicker`, `DateRangePicker`, `DateField`....., etc :)

What is the correct way to do this with React Spectrum? Should I recreate the style using `<View UNSAFE_style={}></View>` ?

**Comment by snowystinger** [comment, marked as answer]
ActionGroup accepts an Aria labelledby https://react-spectrum.adobe.com/react-spectrum/ActionGroup.html#props
And we do have an undocumented Label component. https://www.npmjs.com/package/@react-spectrum/label not sure what our plans are for officially supporting that, but it hasn't changed much. You could use that for now. You'd need to look at the source and stories for how to use it.

**Reply by martpet** [reply]
This is exactly what I needed. Thank you!

**Reply by devongovett** [reply]
Just keep in mind that since it is undocumented and internal, the API can change without warning.
